script.js

```diff
@@ -0,0 +1,115 @@
+import { filmsSet } from "./data/test-data.js";
+
+const header = {
+  css: "header",
```

Comment:
use 'native' webix styling for toolbar instead of custom css
like this: https://snippet.webix.com/stuf64ow

Reply:
oh, thanks,
fixed it.
Made all the changes, you can check again